pkg/hintrunner/zero/zerohint_blake.go

```diff
@@ -118,16 +118,21 @@ func createBlake2sAddUint256Hinter(resolver hintReferenceResolver, bigend bool)
 	return newBlake2sAddUint256Hint(low, high, data, bigend), nil
 }
 
+// Blake2sFinalize hint finalizes the Blake2s hash computation
```

Comment:
should we explain what does  `finalizes the Blake2s hash computation` mean?

Reply:
added some coment